plugins/renamerOnUpdate/README.md

```diff
@@ -26,7 +26,7 @@ Using metadata from your stash to rename your file.
 
 ## Configuration
 
-- Read/Edit `config.py`
+- Read/Edit `config.py` for further information
```

Comment:
either `Read ` config.py` for further information` or leave it as is (edit + for further information doesnt sound good)